snmp/datadog_checks/snmp/data/default_profiles/aruba-wireless-controller.yaml

```diff
@@ -0,0 +1,334 @@
+extends:
+  - _aruba-base.yaml
+  - _base.yaml
+  - _generic-if.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.14823.1.1.1
+  - 1.3.6.1.4.1.14823.1.1.2
+  - 1.3.6.1.4.1.14823.1.1.3
+  - 1.3.6.1.4.1.14823.1.1.4
+  - 1.3.6.1.4.1.14823.1.1.7
+  - 1.3.6.1.4.1.14823.1.1.8
+  - 1.3.6.1.4.1.14823.1.1.9
+  - 1.3.6.1.4.1.14823.1.1.10
+  - 1.3.6.1.4.1.14823.1.1.11
+  - 1.3.6.1.4.1.14823.1.1.12
+  - 1.3.6.1.4.1.14823.1.1.13
+  - 1.3.6.1.4.1.14823.1.1.14
+  - 1.3.6.1.4.1.14823.1.1.15
+  - 1.3.6.1.4.1.14823.1.1.16
+  - 1.3.6.1.4.1.14823.1.1.17
+  - 1.3.6.1.4.1.14823.1.1.18
+  - 1.3.6.1.4.1.14823.1.1.19
+  - 1.3.6.1.4.1.14823.1.1.20
+  - 1.3.6.1.4.1.14823.1.1.23
+  - 1.3.6.1.4.1.14823.1.1.24
+  - 1.3.6.1.4.1.14823.1.1.25
+  - 1.3.6.1.4.1.14823.1.1.26
+  - 1.3.6.1.4.1.14823.1.1.27
+  - 1.3.6.1.4.1.14823.1.1.28
+  - 1.3.6.1.4.1.14823.1.1.29
+  - 1.3.6.1.4.1.14823.1.1.30
+  - 1.3.6.1.4.1.14823.1.1.31
+  - 1.3.6.1.4.1.14823.1.1.32
+  - 1.3.6.1.4.1.14823.1.1.33
+  - 1.3.6.1.4.1.14823.1.1.34
+  - 1.3.6.1.4.1.14823.1.1.35
+  - 1.3.6.1.4.1.14823.1.1.39
+  - 1.3.6.1.4.1.14823.1.1.40
+  - 1.3.6.1.4.1.14823.1.1.41
+  - 1.3.6.1.4.1.14823.1.1.42
+  - 1.3.6.1.4.1.14823.1.1.47
+  - 1.3.6.1.4.1.14823.1.1.48
+  - 1.3.6.1.4.1.14823.1.1.56
+
+metrics:
+  - MIB: WLSX-SYSTEMEXT-MIB
+    symbol:
+      name: cpu.usage
+      OID: 1.3.6.1.4.1.14823.2.2.1.2.1.30.0
+  - MIB: WLSX-SYSTEMEXT-MIB
+    symbol:
+      name: memory.usage
+      OID: 1.3.6.1.4.1.14823.2.2.1.2.1.31.0
+  - MIB: WLSX-SYSTEMEXT-MIB
+    symbol:
+      name: wlsxSysExtPacketLossPercent
+      OID: 1.3.6.1.4.1.14823.2.2.1.2.1.32.0
+      tag: packet_loss
+      TODO_tag: Handle unknown tag `packet_loss` (remove this field once fixed)
+  - MIB: WLSX-SWITCH-MIB
+    table:
+      name: wlsxSysXStorageTable
+      OID: 1.3.6.1.4.1.14823.2.2.1.1.1.10
+    symbols:
+      - name: sysXStorageSize
+        OID: 1.3.6.1.4.1.14823.2.2.1.1.1.10.1.3
+      - name: sysXStorageUsed
+        OID: 1.3.6.1.4.1.14823.2.2.1.1.1.10.1.4
+    metric_tags:
+      - column:
+          name: sysXStorageType
+          OID: 1.3.6.1.4.1.14823.2.2.1.1.1.10.1.2
+        tag: sys_x_storage_type
+        mapping:
+          1: ram
+          2: flash_memory
+      - column:
+          name: sysXStorageName
+          OID: 1.3.6.1.4.1.14823.2.2.1.1.1.10.1.5
+        tag: sys_x_storage_name
+  - MIB: WLSX-SWITCH-MIB
+    table:
+      name: wlsxSwitchAccessPointTable
+      OID: 1.3.6.1.4.1.14823.2.2.1.1.3.3
+    symbols:
+      - name: apChannelNoise
+        OID: 1.3.6.1.4.1.14823.2.2.1.1.3.3.1.13
+      - name: apSignalToNoiseRatio
+        OID: 1.3.6.1.4.1.14823.2.2.1.1.3.3.1.14
```

Comment:
I think it can help to prefix all metrics prefixed with `ap`, `wlan`,  `ha` prefix with additional prefix `wlsx`.

`apChannelNoise` -> `wlsx.apChannelNoise`

`wlanStaTransmitRate ` -> `wlsx.wlanStaTransmitRate `

or maybe `apChannelNoise` -> `wlsxApChannelNoise` (camel case) to be consistent with `wlsxNumOfUsersVPN`

`apChannelNoise` -> `wlsx_switch.apChannelNoise`.

Reply:
I'm not sure I see an added value with adding a prefix in this case

Reply:
I get your concern but we don't have this case atm and we are not sure that it will actually occur so why not handle this in the futur if it happens rather than add prefixes now  ?